recalls-external-api-specification-0.3.0.yml

```diff
@@ -0,0 +1,302 @@
+---
+openapi: 3.0.1
+info:
+  title: DVSA - Vehicle Recalls Service API
+  description: API to allow vehicle manufacturers to manage recording outstanding safety recalls for vehicles with DVSA
+  version: 0.3.0
```

Comment:
please update to 0.3.1

Reply:
file has been replaced with newer version